fork-visualizer/src/app/components/Flow.jsx

```diff
@@ -0,0 +1,60 @@
+"use client"
+
+import { useEffect } from "react"
+import ReactFlow, { useNodesState, useEdgesState, Controls } from 'reactflow';
+import 'reactflow/dist/style.css';
+
+
+import '@/index.css';
+
+
+const connectionLineStyle = { stroke: '#fff' };
+const snapGrid = [20, 20];
+
+const defaultViewport = { x: 0, y: 0, zoom: 10.5 };
+
+const Flow = ({ initialNodes, initialEdges }) => {
+    const [nodes, setNodes, onNodesChange] = useNodesState([]);
+    const [edges, setEdges, onEdgesChange] = useEdgesState([]);
+
+    useEffect(() => {
+        setNodes(initialNodes);
+    }, [initialNodes]);
+
+
+    useEffect(() => {
+        setEdges(initialEdges);
+    }, [initialEdges]);
+
+    return (
+        <ReactFlow
+            nodes={nodes}
+            edges={edges}
+            onNodesChange={onNodesChange}
+            onEdgesChange={onEdgesChange}
+            connectionLineStyle={connectionLineStyle}
+            snapToGrid={true}
+            snapGrid={snapGrid}
+            defaultViewport={defaultViewport}
+            fitView
+            attributionPosition="bottom-left"
+            style={{ background: '#1A192B' }}
+
```

Comment:
```suggestion
```

Reply:
The suggestion is just to remove the blank line, right? Is there some standard tool we should be using here? Like what's the equivalent of `rust-fmt` and `clippy`? I've heard of es-lint before; is that easy to setup, reliable, etc?

Reply:
I have removed this file since it is no longer used